CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Changelog
+
+## v0.0.1 (2017-11-07)
```

Comment:
I'm not sure v0.0.1 is proper version. imo well-tested and/or widely-used plugins can start from v1.0.0.

Reply:
I agree. I will change it.